pkg/asset/appliance/appliance_liveiso.go

```diff
@@ -153,6 +166,41 @@ func (a *ApplianceLiveISO) buildLiveISO(envConfig *config.EnvConfig, applianceCo
 	)
 	spinner.FileToMonitor = consts.DeployIsoName
 
+	// Create bootstrap.img file
+	coreOSConfig := coreos.CoreOSConfig{
+		ApplianceConfig: applianceConfig,
+		EnvConfig:       envConfig,
+	}
+	c := coreos.NewCoreOS(coreOSConfig)
+	ignitionBytes, err := json.Marshal(recoveryIgnition.Bootstrap)
+	if err != nil {
+		logrus.Errorf("Failed to marshal recovery ignition to json: %s", err.Error())
+		return log.StopSpinner(spinner, err)
+	}
+	bootstrapImagePath := filepath.Join(workDir, bootstrapImageName)
+	if err := c.WrapIgnition(ignitionBytes, bootstrapIgnitionPath, bootstrapImagePath); err != nil {
+		logrus.Errorf("Failed to create bootstrap image: %s", err.Error())
+		return log.StopSpinner(spinner, err)
+	}
+
+	// Add bootstrap.img to initrd
```

Comment:
An initramfs file is [defined](https://www.kernel.org/doc/html/latest/driver-api/early-userspace/buffer-format.html) as a concatenation of any number of:

- CPIO archives
- compressed CPIO archives
- the byte `\0`

So if you append the bootstrap image to the initrd, you can avoid modifying the grub config, and thus the kargs info.